docs/content/preview/deploy/multi-dc/async-replication.md

```diff
@@ -132,15 +132,17 @@ Replication lag is computed at the tablet level as follows:
 
 *hybrid_clock_time* is the hybrid clock timestamp on the source's tablet-server, and *last_read_hybrid_time* is the hybrid clock timestamp of the latest record pulled from the source.
 
-An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates the replication lag. The script requires the [`jq`](https://stedolan.github.io/jq/) package.
+An example script `determine_replication_lag.sh` located in the `files` directory calculates the replication lag. The script requires the jq package.
 
-The following example generates a replication lag summary for all tables on a cluster. You can also request an individual table.
+The following example generates a replication lag summary for all tables on a cluster:
 
 ```sh
 ./determine_repl_latency.sh -m 10.150.255.114,10.150.255.115,10.150.255.113
 ```
 
-To obtain a summary of all command options, execute `determine_repl_latency.sh -h` .
+You can also request an individual table.
+
+To obtain information about the overall maximum lag, you should check `/metrics` or `/prometheus-metrics` for `async_replication_sent_lag_micros` or `async_replication_committed_lag_micros` and take the maximum of these values across each source's T-Server.
```

Comment:
Do we want to point to docs on setting up node exporter/prometheus in the first place as well? This subsection is under YB core which means they will have to [set up prometheus manually](https://docs.yugabyte.com/preview/explore/observability/prometheus-integration/linux/).